src/main/resources/org/jenkinsci/plugins/codesonar/CodeSonarPublisher/help-serverCertificateCredentialId.html

```diff
@@ -1 +1,7 @@
-<div>Jenkins Credentials containing HTTPS Server certificate of CodeSonar Hub. Credentials must be of type <i>Secret File</i> and must include a X509 certificate.</div>
\ No newline at end of file
+<div>
+    A copy of the hub's server certificate as a Base-64 encoded ASCII X509 certificate file.
+    <br/>
+    [Optional, HTTPS hubs only] This setting is useful if your hub is not automatically trusted by your Jenkins server, for example, if your hub uses a self-signed certificate.
+    <br/>
+    The certificate file should be provided as a Credential of type <i>Secret File</i>.
+</div>
```

Comment:
[Optional, HTTPS hubs only]
A Credentials parameter of type **secret file**, where the file contains the hub server certificate in PEM format.

This setting is useful if your hub is not automatically trusted by your Jenkins server, for example, if your hub uses a self-signed certificate.

Reply:
Not sure if you intend for us to remove the first sentence.  I think we should be clear about what we mean by "PEM".  PEM is not really a format, and some programs, such as Google Chrome refer to this "PEM" format as "Base-64 encoded ASCII".

I think the entire passage should be:

>CodeSonar Hub server certificate.
>
>[Optional, HTTPS hubs only]
>A Credentials parameter of type secret file, where the file contains the hub server certificate in PEM format (Base-64 encoded ASCII).
>
>This setting is useful if your hub is not automatically trusted by your Jenkins server, for example, if your hub uses a self-signed certificate.

Reply:
ok